app/form_objects/support/privileged_user_form.rb

```diff
@@ -0,0 +1,45 @@
+class Support::PrivilegedUserForm
+  include ActiveModel::Model
+
+  attr_accessor :full_name, :email_address, :privileges
+
+  validates :full_name, presence: { message: 'Enter the name of the user' }
+  validates :email_address, presence: { message: 'Enter the email address of the user' },
+                            format: { with: /\A.*@computacenter.com|.*@digital.education.gov.uk|.*@education.gov.uk\z/,
+                                      message: 'Email address must be end with @computacenter.com, @digital.education.gov.uk or @education.gov.uk' }
+  validate :validate_email_not_taken
+  validate :validate_atleast_one_privilege
+  validate :validate_acceptable_privileges
+
+  def create_user!
+    User.create!(full_name: full_name, email_address: email_address, is_support: is_support?, is_computacenter: is_computacenter?)
+  end
+
+private
+
+  def is_support?
```

Comment:
If these predicate methods were renamed to something like `support?` then we could use RSpec syntactic sugar on the assertions (see other comment in pull request).

Reply:
See my other comment. this is private so damage is limited but can be changed when we tackle the whole problem